http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GKEInterpreter.scala

```diff
@@ -1229,14 +1238,18 @@ class GKEInterpreter[F[_]](
     )
   }
 
-  private[util] def buildCromwellAppChartOverrideValuesString(appName: AppName,
-                                                              cluster: KubernetesCluster,
-                                                              nodepoolName: NodepoolName,
-                                                              namespaceName: NamespaceName,
-                                                              disk: PersistentDisk): List[String] = {
+  private[util] def buildCromwellAppChartOverrideValuesString(
+    appName: AppName,
+    cluster: KubernetesCluster,
+    nodepoolName: NodepoolName,
+    namespaceName: NamespaceName,
+    disk: PersistentDisk,
+    customEnvironmentVariables: Map[String, String]
+  ): List[String] = {
     val proxyPath = s"/proxy/google/v1/apps/${cluster.googleProject.value}/${appName.value}/cromwell-service"
     val k8sProxyHost = kubernetesProxyHost(cluster, config.proxyConfig.proxyDomain).address
     val leoProxyhost = config.proxyConfig.getProxyServerHostName
+    val gcsBucket = customEnvironmentVariables.getOrElse("WORKSPACE_BUCKET", "")
```

Comment:
I don't know whether it makes sense in context, but sometimes I like to do something like
```
customEnvironmentVariables.getOrElse("WORKSPACE_BUCKET", "<no workspace bucket defined>")
```
so that error messages look like
```
cannot access 'gs://<no bucket in config>/blah.txt'
```
instead of
```
cannot access 'gs:///blah.txt'
```

Reply:
Or even better, can we fail here instead of letting cromwell get half-started and then fail with an awkward error message that needs interpretation?